packages/react/src/input/Input.tsx

```diff
@@ -27,11 +27,7 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
         w="256"
         {...ParentBoxrecipe(props)}
       >
-        {leftSection && (
-          <Box >
-            {leftSection}
-          </Box>
-        )}
```

Comment:
I'm not sure what editor setup you're using... but these should never be separate commit. You should always commit linted and formatted codes.

Reply:
Got it vai.